src/Function.cpp

```diff
@@ -122,6 +124,15 @@ InitializeAttributesChoices()
 	sanitize_choices.push_back("pointer-compare");
 	sanitize_choices.push_back("pointer-subtract");
 	sanitize_choices.push_back("leak");
+
+	//pushing optimization options
+	optimize_options.push_back("-O0");
```

Comment:
We have updated the compiler. You should try to rewrite this function with static initializers.

Reply:
Sure. First let's solve changes for alias function attribute.